src/Features/CSharp/Organizing/Organizers/MemberDeclarationsOrganizer.Comparer.cs

```diff
@@ -99,8 +99,8 @@ public int Compare(MemberDeclarationSyntax x, MemberDeclarationSyntax y)
                     return compare;
                 }
 
-                var xName = x.GetNameToken();
-                var yName = y.GetNameToken();
+                var xName = ShouldCompareByName(x) ? x.GetNameToken() : default(SyntaxToken);
+                var yName = ShouldCompareByName(y) ? y.GetNameToken() : default(SyntaxToken);
 
                 if ((compare = TokenComparer.NormalInstance.Compare(xName, yName)) != 0)
```

Comment:
I'm not sure--my only intent is that the new code does what the old code did (after changes to ```GetNameToken```).